Export Dialog styling, no positioning changes

DivyaPrabhakar opened the pull request:
**Things addressed:**

- Added a new class to describe the header type that is seen in the Export Dialog 
- Took out all <Gutter> elements in ExportAllPanel.jsx 
- Corrected the colors for the title header, name of asset, icons, asset export size list, scrollbar
- Aligned the icons with the labels or checkboxes with the labels
- Corrected the button alignment
- Increased the width of the entire dialog so that the spacing between columns looked better and all the asset names were not too cramped

Note @davealonzo the Prefix checkbox is here, so we are good on that front. 

**Things not addressed:**

- Currently I am having trouble with the checkboxes. @davealonzo said that the unchecked ones should only have a stroke around them and then when checked, they should have a background. I am still working on this but I didn't want all the other changes to be held up by this, unless people this is a critical part that must be implemented. @placegraphichere @davealonzo Let me know about this. 

@mcilroyc @designedbyscience @shaoshing You guys looked at my last PR that had some merge conflicts and I addressed all the comments in this new PR. Please let me know if you guys don't see the changes or have any other comments about this. 

Final look of the export dialog: 

![export dialog](https://www.dropbox.com/s/yq1ghww0md3k8h0/Screenshot%202015-09-11%2019.58.24.png?raw=1)

iwehrman commented:
Because we have an intermediate, short-term deadline that involves the UI, I think it is perfectly fine to file a bug for the checkbox styling improvements and address them separately.

mcilroyc commented on src/js/jsx/shared/TitleHeader.jsx:
```diff
@@ -38,6 +39,12 @@ define(function (require, exports, module) {
+                "section-title": true
+            };
+
+            var className = classnames(classNameSet, this.props.className);
```
this is a nit, but to echo @shaoshing's suggestion from the first PR, this would be better as: `var className = classnames("section-title", this.props.className);` and remove the three previous lines.

mcilroyc commented on src/js/jsx/sections/export/ExportAllPanel.jsx:
```diff
@@ -96,17 +95,16 @@ define(function (require, exports, module) {
             return (
                 <div className="exports-panel__layer-wrapper" >
-                    <div className="column-2">
+                    
```
unnecessary newline (not important)

davealonzo commented:
A few design/layout issues remain...

The icons in the asset column don't align vertically. The graphic icons, for example, should be center aligned with the folder icons (also, these icons seem really large compared to the artboard icons – @placegraphichere can you verify correct icon size?)

There are uneven amounts of padding between the left and right panel edges and columns. Left side has more padding than the right.

"Cancel | Export" buttons should be right aligned with the right column margin.

mcilroyc commented:
lookin' good!  I found two very minor nits, but they needn't be fixed in this PR.  I agree with Ian about tackling the checkboxes separately.  I am the one that made the first pass at checkbox styling... so I *might* be able to help with further tweaks.

mcilroyc commented:
@davealonzo quick note/question about the cancel button:  I have a note in a trello card from @placegraphichere saying that we may consider removing the cancel button as it stands now.  I also am aware of a slight shifting of the buttons when "export" turns in to a spinner.  I didn't try hard to fix it yet, because I knew about the possibility of removing Cancel.  Anyway, @DivyaPrabhakar, as part of this PR you can try to fix some aspect of this, or I can do it as a separate PR.

davealonzo commented:
OK, I will let Seth chime in on the fate of the cancel button.

In the meantime, here's an overview of the checkbox UI from the Spectrum style sheet. Not sure we need focus, disabled, or error. But the unchecked state is a 2px stroke thickness.

<img width="1638" alt="screen shot 2015-09-11 at 8 41 13 pm" src="https://cloud.githubusercontent.com/assets/12259315/9829724/4224305e-58c6-11e5-85dc-8ea351667ed7.png">

mcilroyc commented:
I forked this branch and tried to change the checkbox style slightly.  http://cl.ly/image/133E1e2h450j
Note this has a .3rem border, and .2rem was too slim to my eye.

I'm including this comment here just to keep the discussion in one place.

https://github.com/adobe-photoshop/spaces-design/compare/mcilroy/checkbox-unchecked-style?expand=1

DivyaPrabhakar commented:
Update: 

**Changes**

- Fixed the code changes given by @mcilroyc. 
- Fixed the changes given by @davealonzo: 
    1. @placegraphichere confirmed we are taking out the "Cancel" button
    2. Aligned the "Export" button to the right of the second column 
    3. Evened out the padding on the sides of the export dialogue. @davealonzo take a look and let me know if this is better, else I can change it again. 
    4. Changed the icon size and lined up the icons vertically and horizontally. 

**To be changed:** 
@mcilroyc has the changes for the checkboxes so we will work on integrating them today. 

@davealonzo and @iwehrman given the current state of the dialog, as seen below, you can now determine how close we are to nitpicking versus solving UI changes that are unacceptable for the MAX build. Let me know what you guys this. 

Current look of the export dialog: 
![](https://www.dropbox.com/s/rvmw8scd4u4sgt2/Screenshot%202015-09-14%2011.41.25.png?raw=1)

mcilroyc commented:
this is a good merging place.  We shall tackle further tweaks in a separate PR.
